appinfo/info.xml

```diff
@@ -34,7 +34,7 @@ Currently this works for Files, and we have [plans to integrate many more source
     </documentation>
 
     <dependencies>
-        <nextcloud min-version="12" max-version="14"/>
+        <nextcloud min-version="14" max-version="15"/>
```

Comment:
I wouldnt claim that is works with 15 now...

Reply:
Yeah, but the point is that directly after 14 release the main version for devs will be 15, and nothing initially changes. :)